belt-node8-service.gitlab-ci.yml

```diff
@@ -1,5 +1,8 @@
 image: briangweber/node:8
 
+variables:
+  DEFAULT_BRANCH: develop
```

Comment:
master just seems like a much more common convention

Reply:
I guess it is defined as it to not break publish/deploy steps of current morch projects using develop as default branch.